src/twisted/logger/_logger.py

```diff
@@ -19,6 +19,15 @@ class Logger(object):
     A L{Logger} emits log messages to an observer.  You should instantiate it
     as a class or module attribute, as documented in L{this module's
     documentation <twisted.logger>}.
+
+    @type namespace: L{str}
+    @ivar namespace: the namespace for this logger
```

Comment:
we already have a good dosctring from this here https://github.com/twisted/twisted/pull/780/files#diff-88ae4a9d01a009bd424a1764a2ba5319R46

why add a different documentation?